Utilities/src/main/java/edu/stanford/hivdb/mutations/Mutation.java

```diff
@@ -613,4 +608,8 @@ public String getHumanFormatWithoutCons() {
 	public String getHumanFormatWithGene() {
 		return gene.toString() + "_" + getHumanFormat();
 	}
+	
+	public static Pattern getPattern() {
```

Comment:
We need to find a new home for this static method after I changed `Mutation` to interface.